src/interpreter/vm/VM.cpp

```diff
@@ -165,12 +168,25 @@ int VM::run() {
                     exit(EXIT_FAILURE);
                 }
                 break;
+            case Opcode::CNCT:
+                b = pop();
+                a = pop();
+                if (a.type == DurianType::String && b.type == DurianType::String) {
+                    push(DurianObject(a.value.sval.len /* + b.value.sval.len */, a.value.sval.val /* TODO: Implement concat string */));
+                } else {
+                    std::cout << "TypeError: Invalid operand types for &: "
```

Comment:
can we have a helper function for this?

```cpp
    // ...
    ParserErrors::ReportTypeError( /* ... invalid operands or something */ )
    // ...
```

so we can get some standardized printing and management, will make it easier to flesh things out later instead of leaving `std::cout` everywhere